<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>NSBMD Converter</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLineEdit" name="sourceMDL_loc">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>10</y>
      <width>341</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Source Model Location:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="sourceMDLpath_btn">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>10</y>
      <width>93</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Select</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>NSBMD Save Location:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="saveNSBMD_loc">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>50</y>
      <width>341</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="saveNSBMDpath_btn">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>50</y>
      <width>93</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Select</string>
    </property>
   </widget>
   <widget class="QPushButton" name="convert_btn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>260</y>
      <width>581</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Convert and Save</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="cmd_out">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>551</width>
      <height>161</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(0, 255, 0);</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="plainText">
     <string>Select a model file and a NSBMD save path then click convert!</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>90</y>
      <width>21</width>
      <height>161</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
